server/memphis_handlers_configurations.go

```diff
@@ -199,3 +201,13 @@ func changeLogsRetention(logsRetention int) error {
 func (ch ConfigurationsHandler) GetClusterConfig(c *gin.Context) {
 	c.IndentedJSON(200, gin.H{"pm_retention": POISON_MSGS_RETENTION_IN_HOURS, "logs_retention": LOGS_RETENTION_IN_DAYS})
 }
+
+func (s *Server) UpdateClusterAndStationConfigurationsChange(configurationUpdate models.ConfigurationsUpdate) {
```

Comment:
this function aims to update clients regarding all kinds of updates, so please move it to the sdk handler and change the name of it to sendUpdateToClients